quality/metrics/metrics.md

```diff
@@ -0,0 +1,56 @@
+---
+parent: Building and research
+has_children: true
+nav_order: 150
+title: Metrics
+description: Metrics for evaluating machine translation
+---
+
+Metrics are formulas that represent the quality of the machine translation output.
+
+Machine translation quality metrics are divided into string-based metrics and pre-trained metrics.
+
+## String-based metrics
+
+String-based metrics measure the lexical matching between the system output and the reference translation.
+
+### Challenges
+
+String-based metrics are challenging because there are many possible translations for a single source text.
+
+### Examples
+
+- BLEU
+- METEOR
+- NIST
+- chrF
+- TER
+
+*Note: The list is incomplete.*
+
+### Pre-trained metrics
+
+Pre-trained metrics use a language model. These metrics embed the source sentence into a multidimesional space to calculate the differences.
+
+- COMET
+- YiSi
+- BERTscore
+
+*Note: The list is incomplete.*
+
+| String-based metrics | Pre-trained metrics |
+| --- | --- |
+| Support any language | Support only pre-trained language |
+| Need reference | Do not reference |
+| Low correlation with human evaluation | High correlation with human evaluation |
```

Comment:
BERT has high correlation when systems are very bad.

As much as I love tables, I'd probably not do this as a table.

I'd do it like this

> ### Language support
> String-based metrics can support any language.  But the density of words and characters vary by language.  So the accuracy varies by language pair, and the results cannot be compared across language pairs.
>
> Machine learning-based metrics require a model that was trained on data with the source and target languages.
> 
> ### Accuracy
> Machine learning-based metrics have a higher correlation with human evaluation.
> 
> But metrics based on machine learning are not explainable or fair, so they cannot be used in a research competition.